app/views/org_admin/conditions/_webhook_form.html.erb

```diff
@@ -18,7 +18,7 @@
         <div class="md-form mb-5">
           <i class="fa fa-envelope prefix grey-text"></i> <!-- mdbootstrap should have treatment for bad input for this -->
           <%= label_tag name_start + "[webhook-email]", _('Email'), {'data-error': 'wrong', 'data-success': 'right'} %>
-          <%= text_field_tag name_start + "[webhook-email]", nil, placeholder: _('recipient_email@example.com'), class: 'form-control validate' %>
+          <%= email_field_tag name_start + "[webhook-email]", nil, placeholder: _('recipient_email@example.com'), class: 'form-control validate' %>
```

Comment:
in most browsers, using the `email_field_tag` like you have here should be enough. You can also consider wiring in some JS just in case. Maybe something like:
```javascript
import { isUndefined } from '../../../utils/isType';
import { isValidText, isValidEmail } from '../../../utils/isValidInputType';

$(() => {
  const form = $('#my-form-id');

  if (!isUndefined(form)) {
    form.on('submit', (e) => {
      form.find('input[type="email"]').each((i, el) => {
        const value = el.val();
        // First ignore if it was left blank <- remove if you require a value
        if (isValidText(value)) {
          if (!isValidEmail(value)) {
            e.preventDefault();
            return false;
          }
        }
      });
    });
  }
  return true;
});
```